public/css/style.css

```diff
@@ -468,8 +473,12 @@ tr:last-child td {
   height: 56px;
   line-height: 56px;
   padding-left: 20px;
-  display: none;
   pointer-events: none;
+  transform: translateY(56px);
+  opacity: 0;
+  transition-property: transform,opacity;
```

Comment:
`opacity` and `transform` are hardware accelerated. That's why I added both instead of `all`. So we avoid inadvertently animating something that is slow.